py-polars/polars/dataframe/frame.py

```diff
@@ -1285,7 +1285,7 @@ def __dataframe__(
         allow_copy: bool = True,  # noqa: FBT001
     ) -> PolarsDataFrame:
         """
-        Convert to a dataframe object implementing the dataframe interchange protocol.
+        Convert to a DataFrame object implementing the dataframe interchange protocol.
```

Comment:
Can we keep the casing here - object is technically a `PolarsDataFrame` but that's not important here.

```suggestion
        Convert to a dataframe object implementing the dataframe interchange protocol.
```